### DISCUSSION TITLE
Fail to install anvil, chisel

### DISCUSSION THREAD

**Discussion by jessezwd** [foundry-rs/foundry#5502] [Q&A]
According to the docs: "Running foundryup by itself will install the latest (nightly) [precompiled binaries](https://book.getfoundry.sh/getting-started/installation#precompiled-binaries): forge, cast, anvil, and chisel." , 
my result is: 
"$ls .foundry/bin/
cast  forge  foundryup"
foundryup fails to install anvil and chisel, and no any error occurs. what's the problem? how to fix it?

Thanks!

**Comment by Akash-Kolekar** [comment]
You have to first run this command to install foundryup and then follow the instructions on-screen, which will make the foundryup command available in your CLI.
```
curl -L https://foundry.paradigm.xyz | bash
```

**Reply by jessezwd** [reply]
Yes, I did that before. Today I run the following commands again, anvil and chisel are installed.
 curl -L https://foundry.paradigm.xyz | bash
foundryup

I don't know why the problem is solved. Anyway, thanks a lot!
